lang/en.js

```diff
@@ -33,6 +33,7 @@ export default {
 	"components.input-date.errorOutsideRange": "Date must be between {minDate} and {maxDate}",
 	"components.input-date.openInstructions": "Use date format {format}. Arrow down or press enter to access mini-calendar.",
 	"components.input-date.setToToday": "Set to Today",
+	"components.input-number.errorOutsideRange": "Number must be between {minNumber} and {maxNumber}.",
```

Comment:
This could probably just go up with `components.form-element.input.number` as `.rangeError`?